<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>previewSettingsDialog</class>
 <widget class="QDialog" name="previewSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Application to use</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="appNameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Environment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="envList">
           <property name="toolTip">
            <string>Double click on item to edit it.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="envEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addEnvBut">
             <property name="text">
              <string>Add value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeEnvBut">
             <property name="text">
              <string>Remove value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Arguments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="argList">
           <property name="toolTip">
            <string>Double click on item to edit it.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="argEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="addArgBut">
             <property name="text">
              <string>Add value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeArgBut">
             <property name="text">
              <string>Remove value</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="defaultsBut">
         <property name="text">
          <string>Defaults</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>previewSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>previewSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
